setup/tue-functions.bash

```diff
@@ -838,8 +838,9 @@ function tue-get
     shift
 
     #Create btrfs snapshot if possible and usefull:
-    if [[ "$cmd" =~ ^(install|update|remove)$ ]] && { df --print-type / | grep -q btrfs; }
+    if [[ -z "$BTRFS_SNAPSHOT" && "$cmd" =~ ^(install|update|remove)$ ]] && { df --print-type / | grep -q btrfs; }
```

Comment:
Shouldn't this be -n and not -z?

Reply:
@Timple how does this btrfs snapshot work because I have never seen it work? Also the '/' in a standard ubuntu installation is ext4 so the second check in the if block always fails

Reply:
It only works on an BTRFS filesystem (can be chosen when you install ubuntu). It can make instant snapshots which don't take any space until you modify files again.

Reply:
`{ df --print-type / | grep -q btrfs; }` this will be false on your PC then.

Reply:
@Timple how about the `-z` or `-n`?

Reply:
@ar13pit is right, should be -n.

Probably didn't catch is because of the other bug (found by linter) which covered this bug :)

Reply:
```suggestion
    if [[ -n "$BTRFS_SNAPSHOT" && "$cmd" =~ ^(install|update|remove)$ ]] && { df --print-type / | grep -q btrfs; }
```